src/OpenVRTrackersDriver.cpp

```diff
@@ -105,6 +119,11 @@ bool openvr::DevicesManager::initialize()
 
     const auto lock = std::unique_lock(pImpl->mutex);
 
+    if(vrOrigin.compare("standing") == 0) {
```

Comment:
You could simply use
```
vrOrigin == "standing"
```

Also, I would suggest using capital case, as the ``Seated`` case.

Reply:
Sure, done in https://github.com/ami-iit/yarp-openvr-trackers/pull/12/commits/2c1a682b998a0d629495c96f3ded5661eff20e4a